readme.txt

```diff
@@ -93,6 +93,10 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Fix: support custom attributes in Attribute advanced report filter.
 - Fix: Don't show Stock and Reviews Homescreen panels too early.
 
+== 1.8.3 1/5/2021 ==
+
+- Fix: Fix lacking support for Array.prototype.flat #5987
```

Comment:
This won't be for `.flat()` anymore.  Now the included changes are to compile the `debug` module so it can be used in older browsers like IE11.